tmt/steps/execute/internal.py

```diff
@@ -303,6 +304,17 @@ class ExecuteInternalData(tmt.steps.execute.ExecuteStepData):
              mode is not captured, and ``duration`` has no effect.
              """,
     )
+    ignore_duration: bool = field(
+        default=configure_bool_constant(False, 'TMT_PLUGIN_EXECUTE_TMT_IGNORE_DURATION'),
+        option=('--ignore-duration'),
+        is_flag=True,
+        help="""
+             Ignore test duration value and allow test to run forever.
+             Can be set by ``TMT_PLUGIN_EXECUTE_TMT_IGNORE_DURATION=1`` even
+             when step is not specified on the commandline. This environment variable
+             will be replaced by plan.fmf or CLI arguments.
```

Comment:
* Is this field needed? IIUIC, `ExecuteInternalData` inherits from `ExecuteStepData`, including the fields listed there.
* envvar should be passed in via `envvar` parameter - it will make nicer rendering on command line, in HTML docs, everywhere. Click will be able to consume it too, and it will be mentioned in help, highlighted nicely. When it's incorporated in the help text, nothing like that is possible.

Reply:
Took me a while to realize, but why should I add `envvar` when this variable name is produced by step+plugin+option names? Each plugin option should have it, we don't want to extend each field manually ,do we?

Reply:
removed from ExecuteInternalData in 2051375e

Reply:
> Took me a while to realize, but why should I add `envvar` when this variable name is produced by step+plugin+option names? Each plugin option should have it, we don't want to extend each field manually ,do we?

Hmm, good point, that is indeed correct. And it's done like this for plugin options, but not for step options. So I guess it doesn't matter that much for now.